### PR TITLE
groestlcoin: 25.0 -> 26.0

### PR DESCRIPTION
**Pull request by gruve-p**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by gruve-p** [comment]
Ping @SuperSandro2000

File: pkgs/applications/blockchains/groestlcoin/default.nix [hunk 1]
```diff
@@ -51,12 +52,28 @@ stdenv.mkDerivation rec {
...
+  postInstall = ''
+    installShellCompletion --bash contrib/completions/bash/groestlcoin-cli.bash
+    installShellCompletion --bash contrib/completions/bash/groestlcoind.bash
+    installShellCompletion --bash contrib/completions/bash/groestlcoin-tx.bash
+
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-cli.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoind.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-tx.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-util.fish
+    installShellCompletion --fish contrib/completions/fish/groestlcoin-wallet.fish
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/groestlcoin/default.nix:64]:
```suggestion
    for file in contrib/completions/fish/groestlcoin-*.fish; do
      installShellCompletion --fish $file
    done
```
we can shorten this a bit

**Comment by SuperSandro2000** [comment]
23.05 is deprecated, no need to backport to there.

File: pkgs/applications/blockchains/groestlcoin/default.nix (hunk 1 above)
Comment (gruve-p) [reply pkgs/applications/blockchains/groestlcoin/default.nix]: Done

**Comment by SuperSandro2000** [comment]
Please fixup the review commit into the update one otherwise LGTM

**Comment by gruve-p** [comment]
> Please fixup the review commit into the update one otherwise LGTM

Squashed both commits into 1

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**